cloudinary-http42/src/main/java/com/cloudinary/http42/UploaderStrategy.java

```diff
@@ -86,7 +86,7 @@ public Map callApi(String action, Map<String, Object> params, Map options, Objec
             }
         }
 
-        if (file instanceof String && !((String) file).matches("ftp:.*|https?:.*|s3:.*|data:[^;]*;base64,([a-zA-Z0-9/+\n=]+)")) {
+        if (file instanceof String && !((String) file).matches("ftp:.*|https?:.*|s3:.*|gs:.*|data:[^;]*;base64,([a-zA-Z0-9/+\n=]+)")) {
```

Comment:
You should use `StringUtils.isRemoteUrl()` instead of duplicating the code (it was already duplicated, but this is a good chance to fix it)